src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -403,10 +410,11 @@ class certificates extends viewModelBase {
 
     enterReGenerateCertificateMode(itemToReGenerate: unifiedCertificateDefinition) {
         eventsCollector.default.reportEvent("certificates", "re-generate");
-        this.model(certificateModel.generate());
+        this.model(certificateModel.reGenerate());
```

Comment:
can't regenerate method take thumbprint to avoid setting this manually ?

Reply:
done